Description: defines QT_WEBKIT to display correctly documentation in Qt Creator Help plugin.
Author: Fathi Boudra <fabo@debian.org>
Debian-Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=619443
Bug: http://bugreports.qt.nokia.com/browse/QTBUG-13608
---
 Source/WebKit/qt/qt_webkit_version.pri |    1 +
 1 file changed, 1 insertion(+)

--- a/Tools/qmake/qt_webkit.pri
+++ b/Tools/qmake/qt_webkit.pri
@@ -16,6 +16,7 @@
 
 !contains(QT_CONFIG, modular)|contains(QT_ELIGIBLE_MODULES, webkit) {
     QT_CONFIG += webkit
+    DEFINES += QT_WEBKIT
 } else {
     warning("Attempted to include $$QT.webkit.name in the build, but it was not enabled in configure.")
 }
